File: test/fixtures/php-guzzle/multiline_post_with_data.php
<?php
require 'vendor/autoload.php';

use GuzzleHttp\Client;

$client = new Client();

$response = $client->get('http://localhost:28139/echo/html/', [
    'headers' => [
        'Origin'       => 'http://fiddle.jshell.net',
        'Content-Type' => 'application/x-www-form-urlencoded'
    ],
    'form_params' => [
        'msg1' => 'value1',
        'msg2' => 'value2'
    ]
]);

File: test/fixtures/r-httr2/multiline_post_with_data.r
library(httr2)

request("http://localhost:28139/echo/html/") |>
  req_headers(Origin = "http://fiddle.jshell.net") |>
  req_body_form(
    msg1 = "value1",
    msg2 = "value2"
  ) |>
  req_perform()
